tests/grid/tria_signals_01.output

```diff
@@ -0,0 +1,61 @@
+
+DEAL:1d-1d::n_signal_create : 1
+DEAL:1d-1d::n_signal_pre_refinement : 3
+DEAL:1d-1d::n_signal_post_refinement : 3
+DEAL:1d-1d::n_signal_pre_coarsening_on_cell : 2
+DEAL:1d-1d::n_signal_post_refinement_on_cell : 3
+DEAL:1d-1d::n_signal_copy : 1
+DEAL:1d-1d::n_signal_clear : 1
+DEAL:1d-1d::n_signal_any_change : 5
```

Comment:
For `Tria`, yes. The tests refine a hypercube  globally twice and coarsen it globally once, so we have 3 top level refinement signals, 2^d `coarsening_on_cell` signals and (2^d +1) `refinement_on_cell` signals.

For `p::d::Tria`,  not all details are verified because `p::d::Tria::exec_c_and_r` is called repetitively .  However since  `p::d::Tria::exec_c_and_r` calls `Tria::exec_c_and_r` to do underlying works, I believe the result is reasonable. 
And  because we can't insert signals into p4est, the signals on cell for `p::d::Tria` is not totally reliable as we know the p4est  balancing is out of our control.

On the false alarms, it is hard to foresee what will really happen when invoking signal `pre_refinement`, but  maybe we can make `post_refinement` more reliable by changing it into
```
if (mesh_changed)
{
  signals.post_refinement();
}
```